face/PersonDirectory/python/shared_functions.py

```diff
@@ -0,0 +1,195 @@
+import requests, time
+from PIL import Image
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.vision.face import FaceClient
+from azure.ai.vision.face.models import FaceDetectionModel, FaceRecognitionModel
+
+def detect_faces(subscription_key, endpoint, image_path, injection_header=None):

```

Comment:
None of the callers specify injection_header. Is this a miss?

Reply:
You are right. Injection is added.